CHANGELOG.md

```diff
@@ -30,6 +30,10 @@ and this project adheres to
 ### Changed
 
 - cosmwasm-vm: Upgrade Wasmer to 2.2.1.
+- cosmwasm-std: `MockQuerier` now supports adding custom behaviour for handling
+  Wasm queries via `MockQuerier::update_wasm` ([#1050]).
+
+[#1050]: https://github.com/CosmWasm/cosmwasm/pull/1050
```

Comment:
Ah, damn, this was added to the already released 1.0.0-beta7 section. Will fix on main branch.

Reply:
Oh sorry, it's because it's under all other math related stuff I've been adding lately - didn't pay attention to the actual date.
Good catch, couple more things in changelog to shuffle